rtic-macros/CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ For each category, *Added*, *Changed*, *Fixed* add new entries at the top!
 ## [Unreleased]
 
 - Adapt `slic` backends to new version with `mecall`
+- Unstable support for ESP32-C6
```

Comment:
Please put new entries above, so above "- Adapt ..."

Same for all changelog files